### PR TITLE
Link to DB Options page added

### PR DESCRIPTION
**Pull request by rewsmith**
## Description
The current page is missing a link to the Database Options, so this is just a simple fix to add a link.  I also modified `SQL Platforms` to read `SQL Databases` 

## Types of changes
- [ x] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

### CODE REVIEW

File: tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Installing Tyk on Kubernetes requires a multi-node Tyk licence. If you are evalu
 
 This deployment is NOT designed for production use or performance testing. The Tyk Pro Docker Demo is our full, [Self-Managed]({{< ref "/content/tyk-self-managed/install.md" >}}) solution, which includes our Gateway, Dashboard and analytics processing pipeline. 
 
-This demo will run Tyk On-Premises on your machine, which contains 5 containers: Tyk Gateway, Tyk Dashboard, Tyk Pump, Redis and either MongoDB or one of our supported [SQL Platforms] **add database-options link**.
+This demo will run Tyk On-Premises on your machine, which contains 5 containers: Tyk Gateway, Tyk Dashboard, Tyk Pump, Redis and either MongoDB or one of our supported [SQL Databases] (https://tyk.io/docs/tyk-dashboard/database-options/).
```
Comment (letzya) [inline tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]:
```suggestion
This demo will run Tyk On-Premises on your machine, which contains 5 containers: Tyk Gateway, Tyk Dashboard, Tyk Pump, Redis and either MongoDB or one of our supported [SQL Databases] ({{< ref "/tyk-dashboard/database-options/ >}}).
```

Comment (rewsmith) [reply tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]: Sorry, fixed before I saw your suggested change.  To fix I instead removed the space between [text] and (URL), similar to other links on this docs page

Comment (letzya) [reply tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]: we need to use this format `{{< ref "/tyk-dashboard/database-options/ >}}`

Comment (letzya) [reply tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]: you can just commit my suggestion

Comment (letzya) [inline tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]:
```suggestion
This demo will run Tyk On-Premises on your machine, which contains 5 containers: Tyk Gateway, Tyk Dashboard, Tyk Pump, Redis and either MongoDB or one of our supported [SQL Databases] ({{< ref "/tyk-dashboard/database-options/" >}}).
```

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md (hunk 1 above)
Comment (rewsmith) [reply tyk-docs/content/tyk-on-prem/kubernetes-on-windows.md]: @letzya Suggestion made manually since Commit didn't work

**Review by letzya — APPROVED** [review]

**Pull request closed without merging**